medical_pathology/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Dave Lasley <dave@laslabs.com>
```

Comment:
Use the correct OCA header like https://github.com/OCA/maintainer-tools/blob/master/template/module/exceptions.py#L2